src/AliasDefinitionInterface.php

```diff
@@ -1,10 +0,0 @@
-<?php
-
-namespace Interop\Container\Definition;
-
-/**
- * Alias a container entry to another.
- */
-interface AliasDefinitionInterface extends DefinitionInterface, ReferenceInterface
-{
-}
```

Comment:
If I understand correctly to define an alias you would now do per example:

```php
[
  'identifier' => $definition,
  'alias' => new Reference('identifier'),
]
```

Correct?